pokebattle/settings/base.py

```diff
@@ -122,6 +124,15 @@ def base_dir_join(*args):
     base_dir_join('assets'),
 )
 
+# social-auth
+INSTALLED_APPS += ['social_django', ]
```

Comment:
This is not wrong. However, we usually put all the installed apps together, making it easier to identify which modules are installed on the project.

Adding this new lib to new [here](https://github.com/vanessa/pokebattle/blob/8708565ef1103d02debbe8f75d7203a3723a1bcc/pokebattle/settings/base.py#L57), near the other third party apps, would be the "right approach".